### PR TITLE
Explain a little more about Shell.JS

### PR DESCRIPTION
**Pull request by ariporad**
Hi!

Cash looks really cool, thanks for making it!

This PR just adds a little more explination to the Shell.JS section about the difference between ShellJS and Cash.

### CODE REVIEW

**Comment by ariporad** [comment]
cc @nfisher.

**Comment by ariporad** [comment]
cc @nfischer

File: README.md [hunk 1]
```diff
@@ -179,12 +179,25 @@ Shout out to [@aseemk](https://github.com/aseemk) for donating the name.
 
 #### Doesn't Shell.js do this?
 
-No. 
+No.
+ 
+For those who don't know, [Shell.JS](https://github.com/shelljs/shelljs) is an awesome Node package that implements UNIX shell commands programatically in Javascript. Check it out - really. While Shell.JS was tremendously helpful in figuring out how to accomplish Cash, the two do not really conflict.
 
-For those who don't know, [Shell.js](https://github.com/shelljs/shelljs) is an awesome Node app that implements Unix shell commands programatically in Javascript. Check it out - really. While Shell.js was tremendously helpful in figuring out how to accomplish Cash, the two do not really conflict.
+Shell.JS gives the feel of UNIX commands in a code environmenmt, but aims to impliment the commands in a way that makes sense for a javascript library. This means that many commands return JavaScript objects, and some of the rougher and more dangerous edges of bash have been softened a bit.
```
Comment (nfischer) [inline README.md]: s/impliment/implement

**Comment by nfischer** [comment]
For what it's worth, I like it.

Also, @dthree, feel free to open a PR on our project or modify the wiki pointing toward cash. I recently opened the wiki to point toward projects based off shelljs, and it might be good to have a page for other related projects, such as cash.

File: README.md (hunk 1 above)
Comment (ariporad) [inline README.md]: Fantastik Spellir™

Comment (nfischer) [inline README.md]: I would be lost without vim spell checking enabled by default for my markdown files

Comment (ariporad) [inline README.md]: Fixed.

**Comment by r4j4h** [comment]
Thank you, this will be very helpful for people new to either project trying to figure out where they both fit and when they would want to use one or the other!

**Comment by dthree** [comment]
This looks great! Thanks!

**Comment by dthree** [comment]
@nfischer 

> feel free to open a PR on our project or modify the wiki pointing toward cash.

Thank you!

**Comment by dthree** [comment]
Added another link higher-up in the programmatic section:

https://github.com/dthree/cash/commit/63e1a727aac64742f2b4890e16b495970593b78d

**Comment by nfischer** [comment]
:+1:

**Pull request merged**